libs/pipelines/api_pipeline.py

```diff
@@ -179,6 +183,37 @@ def build_counties_timeseries(
     return APIOutput(key, county_api_data, intervention)
 
 
+def build_county_summary_from_model_output(input_dir) -> List[APIOutput]:
+    """Builds lists of counties available from model output.
+
+    Args:
+        input_dir: Input directory.  Should point to county output.
+
+    Returns: List of API Output objects.
+    """
+    found_fips_by_state = defaultdict(set)
+    found_fips = set()
+    for path in pathlib.Path(input_dir).iterdir():
+        if not str(path).endswith('.json'):
+            continue
+
+        state, fips, intervention, _ = path.name.split('.')
+        found_fips_by_state[state].add(fips)
+        found_fips.add(fips)
```

Comment:
this feels a lil ruff don't we have access to this as a var somewhere?

Reply:
I could load all of the data files, but don't need anything inside them so this was the easiest way to get this dat